### DISCUSSION TITLE
allowed-addr-pattern help

### DISCUSSION THREAD

**Discussion by tmcmanamey** [javamelody/javamelody#1174] [General]
I'm  trying to implement allowed-addr-pattern and when I do it never allows any connections whatsoever.  Even if I put *\.*\.*\.* it still doesn't accept any connections and gives the "can't be reached" message. I can reach the monitor fine if I remove the parameter. Any Ideas?
```
<filter>
	<filter-name>javamelody</filter-name>
	<filter-class>net.bull.javamelody.MonitoringFilter</filter-class>
    <init-param>
    <param-name>allowed-addr-pattern</param-name>
    <param-value>"192\.168\.192\.6"</param-value>
	</init-param>
  </filter>
```

**Comment by evernat** [comment]
See https://github.com/javamelody/javamelody/wiki/UserGuide#16-security

And if I read well, you have `"` at the beginning and at the end of the value, which must not be there.
